### DISCUSSION TITLE
[TPFLSH error 16] One or more inputs are out of range:  inputs are within the solid phase

### DISCUSSION THREAD

**Discussion by Erik9705** [usnistgov/REFPROP-issues#620] [Q&A]
Hi all, I'm currently struggling with an application in which I use integration of **REFPROP** in Matlab to update real water properties in real time. 
At a certain point the error 16 message appears and my code can't proceed. By now i'm using **refpropm.m** code (uploaded on Oct. 17, 2013). 
On this page: https://pages.nist.gov/REFPROP-docs/ i found that version 10 suffers from this issue even if properties are stated below the triple point (as in my case).
Has this happened to someone? 
Thanks in advance

**Comment by ianhbell** [comment]
refpropm is deprecated, please switch to the python interface, it is much more feature rich.

**Reply by Erik9705** [reply]
Hi Ian, is it easy to switch if I don't have any knowledge on Python?

**Comment by ianhbell** [comment]
Check it you, you are still writing code in MATLAB, it is only the interface to REFPROP itself that changes

**Reply by Erik9705** [reply]
Thanks
